tests/helpers.py

```diff
@@ -16,7 +16,7 @@
 if sys.version_info < (3, 9):
 
     def randbytes(n: int) -> bytes:
-        return bytearray(RANDOM.getrandbits(8) for _ in range(n))
+        return bytes(bytearray(RANDOM.getrandbits(8) for _ in range(n)))
```

Comment:
```suggestion
        return bytes(RANDOM.getrandbits(8) for _ in range(n))
```

Is there a reason it needs to be put into a `bytearray` and then into `bytes`?

Reply:
> Is there a reason it needs to be put into a `bytearray` and then into `bytes`?

Nothing but absentmindedness, unfortunately.